drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/model/DropInParamsMapper.kt

```diff
@@ -39,4 +43,10 @@ internal class DropInParamsMapper {
     fun getShopperLocale(checkoutConfiguration: CheckoutConfiguration, sessionParams: SessionParams?): Locale? {
         return checkoutConfiguration.shopperLocale ?: sessionParams?.shopperLocale
     }
+
+    private fun getIsRemovingStoredPaymentMethodsEnabled(
+        dropInConfiguration: DropInConfiguration?,
+        sessionParams: SessionParams?
+    ) = dropInConfiguration?.isRemovingStoredPaymentMethodsEnabled
```

Comment:
This code allows the client side configuration to override the sessions configuration, is that intended?